class.jetpack-connection-banner.php

```diff
@@ -25,6 +25,10 @@ static function init() {
 	 */
 	private function __construct() {
 		add_action( 'current_screen', array( $this, 'maybe_initialize_hooks' ) );
+
+		if ( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME' ) ) {
```

Comment:
I gather we're expected to define this constant locally (in wp-config or in a mu-plugin), is that correct?

Reply:
Correct.

Reply:
just  while we  test,  then it should use the info  from the AB test endpoint